### ISSUE TITLE
remove orphan downloads?

### ISSUE THREAD

**Issue by mgaulton** [Saberre/torrent-cleaner#1]
Any thoughts on doing something similar to what you have already but for the actual download folders? i occasionally run across an issue where the download is kept after the torrent is removed from utorrent.

**Comment by Saberre** [comment]
Do you mean that you want to remove downloaded files which are not in uTorrent? Sorry but I don't have any plans to improve this script yet. Maybe you can make a pull request! I think refactoring should be first, indeed :p

**Comment by mgaulton** [comment]
Yes, indeed. I use a php script to scrub and manage labels and sometimes the downloaded files end up orphaned. I'll play and let you know.

**Comment by Saberre** [comment]
Thanks! I'll look forward to it.
